Not working `pnpm create @tanstack/start@latest`

### Which project does this relate to?

Create Tanstack App

### Describe the bug

Run `pnpm create @tanstack/start@latest`:

`node:internal/modules/esm/resolve:275
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/XXX/Library/Caches/pnpm/dlx/e0c15861a1ac713764f472f8072b34d28ad4539adf14c9afc7099d852dedb12f/19c55ab0ff7-992a/node_modules/.pnpm/@tanstack+create@0.61.0_tslib@2.8.1/node_modules/@tanstack/create/dist/utils' imported from /Users/XXX/Library/Caches/pnpm/dlx/e0c15861a1ac713764f472f8072b34d28ad4539adf14c9afc7099d852dedb12f/19c55ab0ff7-992a/node_modules/.pnpm/@tanstack+create@0.61.0_tslib@2.8.1/node_modules/@tanstack/create/dist/file-helpers.js
    at finalizeResolution (node:internal/modules/esm/resolve:275:11)
    at moduleResolve (node:internal/modules/esm/resolve:860:10)
    at defaultResolve (node:internal/modules/esm/resolve:984:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:719:12)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:643:25)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:626:38)
    at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:279:38)
    at ModuleJob._link (node:internal/modules/esm/module_job:136:49) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///Users/XXX/Library/Caches/pnpm/dlx/e0c15861a1ac713764f472f8072b34d28ad4539adf14c9afc7099d852dedb12f/19c55ab0ff7-992a/node_modules/.pnpm/@tanstack+create@0.61.0_tslib@2.8.1/node_modules/@tanstack/create/dist/utils'
}

Node.js v23.7.0`

### Your Example Website or App

Nothing

### Steps to Reproduce the Bug or Issue

1. Run `pnpm create @tanstack/start@latest`

### Expected behavior

Run

### Screenshots or Videos

_No response_

### Platform

- OS: macOS(zsh)
- Browser: Not related
- Version: 0.59.2(@tanstack/create-start)

### Additional context

_No response_

--- a/packages/create/src/file-helpers.ts
+++ b/packages/create/src/file-helpers.ts
@@ -4,8 +4,8 @@ import { basename, extname, resolve } from 'node:path'
 import parseGitignore from 'parse-gitignore'
 import ignore from 'ignore'
 
-import { hasDrive, stripDrive } from './utils'
-import type { Environment } from './types'
+import { hasDrive, stripDrive } from './utils.js'
+import type { Environment } from './types.js'
 
 const BINARY_EXTENSIONS = ['.png', '.jpg', '.jpeg', '.gif', '.svg', '.ico']
 
@@ -232,10 +232,10 @@ const PROJECT_FILES = ['package.json']
 export function createIgnore(path: string, includeProjectFiles = true) {
   const ignoreList = existsSync(resolve(path, '.gitignore'))
     ? (
-        parseGitignore(
-          readFileSync(resolve(path, '.gitignore')),
-        ) as unknown as { patterns: Array<string> }
-      ).patterns
+      parseGitignore(
+        readFileSync(resolve(path, '.gitignore')),
+      ) as unknown as { patterns: Array<string> }
+    ).patterns
     : []
   const ig = ignore().add(ignoreList)
   return (filePath: string) => {

